package.json

```diff
@@ -25,8 +25,7 @@
     "lint": "eslint --cache . && tslint --project .",
     "pretest": "npm run lint && npm run build",
     "test": "mocha",
```

Comment:
Wait, wait, wait, `test` no longer runs `tsc`????

Reply:
`npm test` runs the pretest script too

Reply:
Okay, it does, `npm test` is different from `npm run test`.

Reply:
You are correct, it does run the `pretest` script.